server/src/integration/java/org/opentosca/toscana/IntegrationTest.java

```diff
@@ -0,0 +1,4 @@
+package org.opentosca.toscana;
+
+public interface IntegrationTest {
```

Comment:
Why is this required?

Reply:
This interface and the UnitTest one represents a [Test category in JUnit](https://github.com/junit-team/junit4/wiki/categories)